prisma/schema/position.prisma

```diff
@@ -1,7 +1,7 @@
 model Position {
   id                Int                @id @default(autoincrement())
   name              String             @unique
-  topIssue          TopIssue?          @relation(fields: [topIssueId], references: [id])
+  topIssue          TopIssue?          @relation(fields: [topIssueId], references: [id], onDelete: Cascade)
```

Comment:
I'm not sure we want this... if we delete a `TopIssue` we want to delete the `Position`s that relate to it.. but if we delete one position, we don't wanna delete the TopIssue associated with it, right?

Reply:
It's super weird in prisma, it works opposite what you'd expect. Notice top issue model doesn't contain any onDelete: cascade statement. This only only delete position if the top issue is deleted, not the reverse. But thinking about this made me realize we should delete campaignPositions if a position is deleted, so I added that.

https://www.prisma.io/docs/orm/prisma-schema/data-model/relations/referential-actions